src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/Common/Invoice.cs

```diff
@@ -50,5 +50,11 @@ public class Invoice
 
         [JsonPropertyName("evt_form_number")]
         public string? FormNumber { get; set; }
+
+        [JsonPropertyName("act")]
+        public string? Act { get; set; }
+
+        [JsonPropertyName("sectionNumber")]
```

Comment:
This property name does not match the one in the mapper and csv file. Should we change this to only `section` or update everything to `section_number`?

Reply:
Let me go thru my thinking and see if I have missed something or introduced a problem

1. These property names from the the RSI service. So the `JsonPropertyName` must be these. This handles the live RSI service deserialization.
2. In the mapper, we map the column to the property [here](https://github.com/bcgov/jag-traffic-courts-online/blob/334537a32c7f9b8e9aa45fa66e2fd550172bd8d6/src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/Mock/MockTicketSearchService.cs#L141). The column name in the csv matches the property name but lower case. We could make them consistent.  It is too bad the RSI team did not keep their naming convention. Below is an example they provided. That being said, I can see the argument of changing the CSV column `section` to `sectionNumber` so the data in the CSV matches properties exactly as returned by the service.

```json
{
  "invoice_number" : "EZ020052841",
  "pbc_ref_number" : "10006", 
  "party_number" : "n/a",
  "party_name" : "n/a",
  "account_number" : "n/a",
  "site_number" : "0",
  "cust_trx_type" : "Traffic Violation Ticket",
  "term_due_date" : "2023-02-24T06:31",
  "total" : 109.0,
  "amount_due" : 109.0,
  "attribute1" : "NOT TO DRIVE IN HOV LANE",
  "attribute2" : "LAMBOTHER",
  "attribute3" : "2023-02-24",
  "attribute4" : "n/a",
  "evt_form_number" : "MV6000E (102018)",
  "act" : "MVA",
  "sectionNumber" : "152"
}
```

Reply:
Ah that makes sense. So we can't change it because it has to match the property name in RSI Service. Yeah, I wish RSI kept the same naming convention. Thanks, looks good to go then.

Reply:
Your logic seems good. I think this is correct. Invoice.cs is an RSI data structure so the json name must be "sectionNumber" in order to deserialize data from RSI.

Reply:
I just pushed a change the csv to use the same column/property names as the RSI service.